tests/conftest.py

```diff
@@ -240,6 +240,7 @@ def teardown():
                 env={"PULL_CACHE": "yes", "AGENT_IMAGE_NAME": agent_image_name, "AGENT_VERSION": agent_image_tag},
                 stderr=subprocess.STDOUT,
                 check=True,
+                cwd=os.path.join(os.path.dirname(os.path.realpath(__file__)), ".."),
```

Comment:
Can you pull this out to a global constant called something like `REPO_ROOT_DIR` or something so that it is more explicit what it is for?